internal/abci/cometbft/node.go

```diff
@@ -253,3 +254,7 @@ func PubkeyToAddr(pubkey []byte) (string, error) {
 	publicKey := cometEd25519.PubKey(pubkey)
 	return publicKey.Address().String(), nil
 }
+
+func (n *CometBftNode) LocalClient() *local.Local {
+	return local.New(n.Node)
+}
```

Comment:
Looks like you didn't go this route?  `migrator.SetCometClient` is commented out in build.go

Reply:
AH, forgot to remove all these